test/tests/pbs_smoketest.py

```diff
@@ -1487,6 +1487,8 @@ def test_import_pbs_ifl_module(self):
         PBS_EXEC/{lib,lib64}/python/altair.
         """
         import pbs_ifl
+        sys.path.append(os.path.join(
+            self.server.pbs_conf['PBS_EXEC'], 'lib','python', 'altair'))
```

Comment:
The 'import pbs_ifl' needs to go after the ..sys.append() otherwise we'll get an import error.